src/handler/archive.ts

```diff
@@ -0,0 +1,78 @@
+import 'dotenv/config';
+import { APIGatewayProxyEvent, APIGatewayProxyResult } from 'aws-lambda';
+import logger from '../util/logger';
+import { validateArchiveErrors } from '../validators/archive';
+import { addHttpHeaders } from '../util/httpHeaders';
+import { archiveRecord, getBySystemNumberAndCreatedTimestamp } from '../services/database';
+import { getUserDetails } from '../services/user';
+import { formatTechRecord } from '../util/formatTechRecord';
+import { ArchiveRecord, ArchiveRecordRequestBody } from '../models/archive';
+
+export const handler = async (event: APIGatewayProxyEvent): Promise<APIGatewayProxyResult> => {
+  logger.info('Archive end point called');
+
+  const archiveErrors = validateArchiveErrors(event);
+
+  if (archiveErrors) {
+    return addHttpHeaders(archiveErrors);
+  }
+
+  try {
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+    const body: ArchiveRecordRequestBody = await JSON.parse(event.body as string);
```

Comment:
You could do the following to avoid switching off ESLint
```suggestion
    const body: ArchiveRecordRequestBody = await JSON.parse(event.body as string) as ArchiveRecordRequestBody;
```